src/blockynumbersaver.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef BLOCKYNUMBERSAVER_HPP
+#define BLOCKYNUMBERSAVER_HPP
+
+#include "blockynumber.hpp"
+
+class BlockyNumberSaver
+{
+private:
+    int index;
+    BlockyNumber* values;
```

Comment:
maybe use vector here?

Reply:
the array is later initalized with a fixed size

Reply:
You could use the [resize](http://www.cplusplus.com/reference/vector/vector/resize/) method of vector. Since vector automatically manages deallocation when it is out of scope, it would be safer to use.

Reply:
I guess so, will do